packages/wct-mocha/src/config.ts

```diff
@@ -17,59 +17,34 @@ export interface Config {
    *
    * Paths are relative to `scriptPrefix`.
    */
-  environmentScripts: string[];
-  environmentImports: string[];
+  environmentScripts?: string[];
+  environmentImports?: string[];
   /** Absolute root for client scripts. Detected in `setup()` if not set. */
-  root: null|string;
+  root?: null|string;
   /** By default, we wait for any web component frameworks to load. */
-  waitForFrameworks: boolean;
+  waitForFrameworks?: boolean;
   /**
    * Alternate callback for waiting for tests.
    * `this` for the callback will be the window currently running tests.
    */
-  waitFor: null|Function;
+  waitFor?: null|Function;
```

Comment:
I did this initially to support partial config objects for the override arguments, not knowing at the time about the `Partial` type.  I've since reverted the making of these as optional properties.